facilitator/src/bin/facilitator.rs

```diff
@@ -705,15 +731,47 @@ fn main() -> Result<(), anyhow::Error> {
     result
 }
 
+fn get_ecies_public_key(
```

Comment:
I was thinking of potentially turning the input of this into a short lived struct and make this a method on it. Not sure if that would be cleaner than this.

Reply:
This seems essentially consistent with other functions like `intake_task_queue_from_args` or `public_key_map_from_arg`.